conversation_reconstruction/dataflow/write_page_states_to_spanner/dataflow_main.py

```diff
@@ -101,17 +123,19 @@ def process(self, element, instance_id, database_id, table_id, table_columns):
   parser.add_argument('--spanner_table',
                       dest='spanner_table',
                       help='The id of the Spanner table.')
-  parser.add_argument('--spanner_table_columns',
-                      dest='spanner_table_columns',
-                      type=list,
+  parser.add_argument('--spanner_table_columns_config',
+                      dest='config_file',
                       default=None,
-                      help='(Optional) The columns in the spanner table.')
+                      help='The config file in json format to specify columns.')
   parser.add_argument('--testmode',
                       dest='testmode',
                       action='store_true',
                       default=None,
                       help='(Optional) Run the pipeline in testmode.')
 
   known_args, pipeline_args = parser.parse_known_args()
+  if known_args.config_file is not None:
+    with open(known_args.config_file, "r") as f:
```

Comment:
If you use `tf.gfile.open` then you can allow filepaths to be cloud storage, or local storage.

Reply:
Is there a way instead of using tensorflow to do it? it feels like adding extra complexity to install tensorflow package to open a file.